website_event_sale_hide_ticket/templates/event_ticket.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template
+        id="registration_template"
+        inherit_id="website_event_sale.registration_template"
+    >
+        <xpath expr="//t[@t-set='tickets']" position="attributes">
+            <attribute
+                name="t-value"
+            >event.event_ticket_ids.filtered(lambda t: not t.is_expired and t.show_in_website)</attribute>
```

Comment:
Isn't it possible to do this with `ir.rules` instead (the same way it's done for events) ?

See: https://github.com/odoo/odoo/blob/13.0/addons/website_event/security/event_security.xml

The downside of doing it with `ir.rules` is that the Odoo users will still be able to see these tickets on the website, which may be confusing, so it has to go with a qweb template that shows admins that the ticket is not published. 

See: https://github.com/odoo/odoo/blob/72cd8d0769097b21bbf94a34496888617058cb1b/addons/website_event/views/event_templates.xml#L199-L201 for example 

But in general it's a more solid approach IMO